kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriter.java

```diff
@@ -69,10 +72,11 @@ public BigQueryWriter(int retries, long retryWaitMs) {
    * @param tableId The PartitionedTableId.
    * @param rows The rows to write.
    * @param topic The Kafka topic that the row data came from.
+   * @return the list of failed rows in partial failure.
```

Comment:
map from failed row id to the BigQueryError

Reply:
good catch